tests/console/force_cron_run.pm

```diff
@@ -30,14 +30,17 @@ sub settle_load {
 
 sub run {
     select_console 'root-console';
-
     # show dmesg output in console during cron run
     assert_script_run "dmesg -n 7";
 
     # Make sure there's no load before we trigger one via cron.
     settle_load;
     my $before = time;
-    assert_script_run "bash -x /usr/lib/cron/run-crons", 1000;
+    #"bash -x /usr/lib/cron/run-crons" doesn't work really on a running system with online duration shorter than 15 minutes
+    # so we can call daily cron jobs explicitly here, no worries about other cron jobs for hourly,weekly,monthly because they
+    # are not available at that time
+    my $run_daily_cronjobs = 'for file in /etc/cron.daily/*; do echo execute daily cron jobs: $file; $file; done';
+    assert_script_run "$run_daily_cronjobs", 1000;
```

Comment:
No need for these quotes. It's just a variable.

But you need to be careful for the case when there are no cron.daily scripts installed. It would try to execute * - so either test -x $file before executing it in the loop. Or change the glob to e.g. for file in $(ls -1 /etc/cron.daily); /etc/cron.daily/$file; done